### PR TITLE
DM-4707: Adapt CFHT astrometry test for DECam COSMOS field validation

### PR DESCRIPTION
**Pull request by wmwv**

### CODE REVIEW

File: runDecamTest.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+echo "Ingesting Raw data"
+INPUT=${WORKSPACE}/input
+mkdir -p $INPUT
+OUTPUT=${WORKSPACE}/output
+mkdir -p $OUTPUT
+
+echo "lsst.obs.decam.DecamMapper" > ${INPUT}/_mapper
+
+RAWDATA=${VALIDATION_DATA_DECAM_DIR}
+ingestImagesDecam.py ${INPUT} ${RAWDATA}/instcal/*.fz --mode link
```
Comment (timj) [inline runDecamTest.sh]: `${RAWDATA}` needs to be quoted. Consider running `shellcheck` on the shell scripts.

**Comment by timj** [comment]
Can you run `flake8` or something on the python code? There is some trailing whitespace, unused imports and a couple of long lines.

**Comment by wmwv** [comment]
Thanks for the nudge to do that.  flake8 and shellcode pass

The various *Config.py files aren't exactly Python source files so I didn't fuss with them.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,48 @@
...
+A set of utilities to run the processCcd task on some 
+CFHT data and DECam data
 and validate the astrometry of the results.
 
 Pre-requisites: install and declare the following
 1. pipe_tasks from the LSST DM stack (note that pipe_tasks is included with lsst_apps, which is the usual thing to install)
 2. obs_cfht from https://github.com/lsst/obs_cfht (this package is not included with lsst_apps); declare this with tag "current"
-3. validation_data_cfht from https://github.com/wmwv/validation_data_cfht
+3. validation_data_cfht from https://github.com/lsst/validation_data_cfht
+4. validation_data_decam from https://github.com/lsst/validation_data_decam
```
Comment (hsinfang) [inline README.md:23]: Also want obs_decam here?

Comment (wmwv) [inline README.md:23]: Yes, thank you.  I rewrote this section a bit now that `obs_cfht` and `obs_decam` are in `lsstsw/etc/repos.yaml`.

File: runCfhtTest.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+INPUT=${WORKSPACE}/input
+mkdir -p $INPUT
+OUTPUT=${WORKSPACE}/output
+mkdir -p $OUTPUT
+
+echo "lsst.obs.cfht.MegacamMapper" > ${INPUT}/_mapper
+
+# ingest CFHT raw data
+RAWDATA=${VALIDATION_DATA_CFHT_DIR}
+ingestImages.py ${INPUT} "${RAWDATA}/raw/*.fz" --mode link
```
Comment (hchiang2) [inline runCfhtTest.sh]: It doesn't work on my machine if *.fz is inside the quotes

File: runDecamTest.sh [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+echo "running processCcd"
+MACH=$(uname -s)
+if [ "$MACH" == Darwin ]; then
+    NUMPROC=$(sysctl -a | grep machdep.cpu | grep core_count | cut -d ' ' -f 2)
+else
+    NUMPROC=$(grep -c processor /proc/cpuinfo)
+fi
+NUMPROC=$((NUMPROC<8?NUMPROC:8))
+
+processCcdDecam.py ${INPUT} --output ${OUTPUT} @runDecam.list --configfile decamConfig.py --clobber-config -j $NUMPROC
```
Comment (hsinfang) [inline runDecamTest.sh:38]: Consider saving the log for future analysis or ease to debug? 
For example --logdest

File: runDecamTest.sh [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+echo "Ingesting Raw data"
+INPUT=${WORKSPACE}/input
+mkdir -p $INPUT
+OUTPUT=${WORKSPACE}/output
+mkdir -p $OUTPUT
+
+echo "lsst.obs.decam.DecamMapper" > ${INPUT}/_mapper
+
+RAWDATA=${VALIDATION_DATA_DECAM_DIR}
+ingestImagesDecam.py ${INPUT} "${RAWDATA}/instcal/*.fz" --mode link
```
Comment (hsinfang) [inline runDecamTest.sh:23]: The ingest doesn't work properly on my machine if *.fz is inside the quotes

**Pull request merged**